site/tutorials/keptn-litmus-08.md

```diff
@@ -0,0 +1,357 @@
+    ```
+
+1. Now that we have added our SLIs, let us add the quality gate in terms of an `slo.yaml` which adds objectives for our metrics that have to be satisfied. earn more about the concept of [Service-Level Objectives in the Keptn docs](https://keptn.sh/docs/concepts/quality_gates/#what-is-a-service-level-objective-slo).
+
+    ```
+    keptn add-resource --project=litmus --stage=chaos --service=helloservice --resource=helloservice/slo.yaml --resourceUri=slo.yaml
+    ```
+
+We've now added our quality gate, let's move on to add the chaos instructions and then run our experiment!
+
+## Adding Litmus Chaos Experiment to Keptn
+Duration: 1:00
+
+We have already installed LitmusChaos on our Kubernetes cluster, but we have not yet added or executed a chaos experiment. Let's do this now!
+
+Let us add the `experiment.yaml` file that holds the chaos experiment instructions. It will be picked up by the LitmusChaos integration of Keptn each time a test is triggered. Therefore, Keptn makes sure that both, JMeter tests as well as LitmusChaos tests, are executed during the `test` task sequence.
+
+```
+keptn add-resource --project=litmus --stage=chaos --service=helloservice --resource=./litmus/experiment.yaml --resourceUri=litmus/experiment.yaml
+```
+
+Great job - the file is added and we can move on!
+
+## Configure Prometheus
+Duration: 2:00
+
+Before we are going to run the experiment, we have to make sure that we have some observability software in place that will actually monitor how the service is behaving under the testing conditions. 
+
+1. Let's use the Keptn CLI to configure Prometheus. It will set up a Prometheus deployment and configures it to be ready for Keptn usage. 
+
+    ```
+    keptn configure monitoring prometheus --project=litmus --service=helloservice
+    ```
+
+1. Next, we are going to add a `blackbox-exporter` for Prometheus that is able to observe our service under test from the outside, i.e., as a blackbox. 
+
+    ```
+    kubectl apply -f ./prometheus/blackbox-exporter.yaml
+    kubectl apply -f ./prometheus/prometheus-server-conf-cm.yaml -n monitoring
+    ```
+
+1. Finally, restart Prometheus to pick up the new configuration
+    
+    ```
+    kubectl delete pod -l app=prometheus-server -n monitoring
+    ```
+
+Now everything is in place, let's run our experiments and evaluate the resilience of our demo application!
+
+## Run experiment
+Duration: 4:00
+
+We are now ready to kick off a new deployment of our test application with Keptn and have it deployed, tested, and evaluated.
+
+1. Let us now trigger the deployment, tests, and evaluation of our demo application.
+
+    ```
+    keptn trigger delivery --project=litmus --service=helloservice --image=jetzlstorfer/hello-server:v0.1.1
+    ```
+
+1. Let's have a look in the Keptn's bridge what is actually going on. We can use this helper command to retrieve the URL of our Keptn's bridge.
```

Comment:
Please change: Keptn`s bridge to --> Keptn Bridge.

(Some time ago, I got feedback from a technical writer that the `s should not be used.)